<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>636</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsViewImageQuery">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsViewImageComplete">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelQuery">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelBase">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbrir">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpenAbrirBase"/>
    <addaction name="actionOpenAbrirQuery"/>
    <addaction name="separator"/>
    <addaction name="actionOpenProcess"/>
   </widget>
   <addaction name="menuAbrir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenAbrirBase">
   <property name="text">
    <string>Image Base</string>
   </property>
  </action>
  <action name="actionOpenCompare">
   <property name="text">
    <string>New Image</string>
   </property>
  </action>
  <action name="actionSIFT_withFLANN">
   <property name="text">
    <string>SIFT + FLANN</string>
   </property>
  </action>
  <action name="actionOBR">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OBR</string>
   </property>
  </action>
  <action name="actionSIFT">
   <property name="text">
    <string>SIFT</string>
   </property>
  </action>
  <action name="actionSURF">
   <property name="text">
    <string>SURF</string>
   </property>
  </action>
  <action name="actionCheckFlannMatcher">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flann Matcher</string>
   </property>
  </action>
  <action name="actionOpenAbrirQuery">
   <property name="text">
    <string>Image Query</string>
   </property>
  </action>
  <action name="actionOpenProcess">
   <property name="text">
    <string>Process Options</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
